packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2264,6 +2264,8 @@ export class StripeHelper {
 
     const payment_provider = this.getPaymentProvider(customer);
 
+    const showPaymentMethod: boolean = !!invoiceTotalInCents;
```

Comment:
nit: your call here on style/readability/debugability, since this is already a primitive value in the scope of this method and we only need this variable in the return object/once, could we change the return object to
```
return {
    <all the other properties>,
    showPaymentMethod: !!invoiceTotalInCents
}
```
and remove the this assignment?

Reply:
Sweet good idea. Made the change!